### PR TITLE
Timezone all database timestamp fields

### PR DESCRIPTION
**Pull request by OxygenCobalt**
This PR gives all database fields and explicit timezone and forces all routes to work with UTC timezones only. Any routes that work with timestamps also now communicate solely with epoch timestamps to prevent having any timezones in transit.

### CODE REVIEW

**Review by ezrichards — APPROVED** [review]
Fine with me -- assuming this is thoroughly tested!! :)

File: backend/src/handlers/alert.py [hunk 1]
```diff
@@ -55,25 +55,25 @@ def get_alert(
...
     return JSONResponse(content=filtered_alert)
 
 
 @router.post("/")
 def post_alert(req: Request, alert_model: AlertModel) -> JSONResponse:
     with req.app.state.db.session() as session:
-        dt_start_time = datetime.datetime.fromtimestamp(alert_model.start_time)
-        dt_end_time = datetime.datetime.fromtimestamp(alert_model.end_time)
+        dt_start_time = datetime.fromtimestamp(alert_model.start_time, timezone.utc)
+        dt_end_time = datetime.fromtimestamp(alert_model.end_time, timezone.utc)
```
Comment (tbwrigh) [inline backend/src/handlers/alert.py:76]: correct me if I'm wrong but it looks like we are assuming that time zone is always sent UTC, should we consider supporting passing a timezone in alert model? If so, I don't think it necessarily needs to be in this ticket but we should potentially make a new issue for it.

File: backend/src/handlers/analytics.py [hunk 2]
```diff
@@ -29,9 +29,9 @@ async def post_ridership_stats(req: Request, van_id: int):
     # Unpack the byte body sent by the hardware into their corresponding values
     body = await req.body()
     timestamp, entered, exited, lat, lon = struct.unpack("!lbbdd", body)
-    timestamp = datetime.fromtimestamp(timestamp)
+    timestamp = datetime.fromtimestamp(timestamp, timezone.utc)
 
-    current_time = datetime.now()
+    current_time = datetime.now(timezone.utc)
```
Comment (tbwrigh) [inline backend/src/handlers/analytics.py:34]: same as alerts.py

**Review by tbwrigh — APPROVED** [review]

File: backend/src/handlers/alert.py (hunk 1 above)
Comment (OxygenCobalt) [reply backend/src/handlers/alert.py:76]: It's a unix timestamp, so it should not really care about timezones. We're just interpreting it as UTC so postgres stores a timezone.

**Pull request merged**